src/main/java/com/hillel/items_exchange/model/Location.java

```diff
@@ -23,7 +23,7 @@
 
     @Enumerated(EnumType.STRING)
     @Column(name = "lang")
```

Comment:
It's looks like you missed rename it into Column annotation.

Reply:
No, I have not changed the column name in the DB. We did not discuss this. Is it critical? I can do it, but you will need to remove the DB one more time.

Reply:
Difference between names creates misunderstanding - you should update it or rename field name to lang.

Reply:
Ok

Reply:
Done